variants/frontend-stimulus-typescript/.eslintrc.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/** @type {import('eslint').Linter.Config} */
+const config = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: 'tsconfig.json',
+    ecmaVersion: 2019,
```

Comment:
We set `ecmaVersion` to `2022` in `variants/frontend-stimulus/.eslintrc.js` here:

https://github.com/ackama/rails-template/blob/f69989ff72c4bf7329b76a8889a675e550d42b6f/variants/frontend-stimulus/.eslintrc.js#L8

Should this also be `2022`?

Reply:
maybe...? The reason we have this is for `static` assignments:

```
import { Controller } from "stimulus"

export default class extends Controller {
  static targets = [ "name", "output" ] // <- this was intro'd in EMCA 2022

  greet() {
    this.outputTarget.textContent =
      `Hello, ${this.nameTarget.value}!`
  }
}
```

I think we've not had errors on it before because TypeScript/babel transpile it away, and that means this isn't relevant because `@typescript-eslint` favors TypeScripts behaviour over `ecmaVersion`?

I'll try to look into it further if/when I have some time, but I don't _think_ it should be a blocker unless someone actually hits errors from it.